geomstats/geometry/dirichlet_distributions.py

```diff
@@ -11,15 +11,15 @@
 import geomstats.backend as gs
 import geomstats.errors
 from geomstats.algebra_utils import from_vector_to_diagonal_matrix
-from geomstats.geometry.embedded_manifold import EmbeddedManifold
+from geomstats.geometry.embedded_manifold import OpenSet
```

Comment:
Yes, or we could have a base.py with all abstract classes?

Reply:
Yes, I like `base.py`: it will allow to describe the 3 base cases of abstract manifolds classes at once, and refer to your PhD thesis.

We should make a note that "base" does not refer to the base manifold of a fiber bundle... x)